extras/startup.bsh

```diff
@@ -49,7 +48,7 @@ print(p + " = " + getprop(p));
 }
 
 loglevel(String priority, String category){
-LoggingManager.setPriority(priority, category);
+// TODO: implement?
```

Comment:
Can we do the TODO before merging, or put the current behaviour back?

Reply:
Pull requests are welcome.
I have no idea what this code means, however, I am 100% sure the previous implementation was doing absolutely nothing since `LogginManager.setPriority` was an empty method.

Reply:
Understood. Could we just remove the TODO? Or even better state this has always been unimplemented?